.circleci/workflows.yml

```diff
@@ -2852,7 +2853,7 @@ linux-x64-workflow: &linux-x64-workflow
           - test-runner:build-binary
           - publish-binary
         requires:
-          - build
+          - node_modules_install
```

Comment:
is it worth mentioning we don't need to wait for the build to complete because the binary job does its own build?

Reply:
i think its okay without. reading the job steps gives insights that we kick off a downstream workflow